src/components/Tree/Tree.tsx

```diff
@@ -0,0 +1,84 @@
+import * as _ from 'lodash'
+import * as PropTypes from 'prop-types'
+import * as React from 'react'
+
+import TreeItem from './TreeItem'
+import { UIComponent, childrenExist } from '../../lib'
+import { ShorthandValue, ShorthandRenderFunction } from '../../../types/utils'
+import { Accessibility } from '../../lib/accessibility/types'
+import { defaultBehavior } from '../../lib/accessibility'
+import { commonUIComponentPropTypes, childrenComponentPropTypes } from '../../lib/commonPropTypes'
+import { UIComponentProps, ChildrenComponentProps } from '../../lib/commonPropInterfaces'
+import * as customPropTypes from '../../lib/customPropTypes'
+
+export interface TreeProps extends UIComponentProps<any, any>, ChildrenComponentProps {
+  /**
+   * Accessibility behavior if overridden by the user.
+   * @default defaultBehavior
+   */
+  accessibility?: Accessibility
+
+  /** Shorthand array of props for Tree. */
+  items: ShorthandValue[]
+
+  /** Whether the tree is a subtree. */
+  nested?: boolean
+
+  /**
+   * A custom render function for the title slot.
+   *
+   * @param {React.ReactType} Component - The computed component for this slot.
+   * @param {object} props - The computed props for this slot.
+   * @param {ReactNode|ReactNodeArray} children - The computed children for this slot.
+   */
+  renderItemTitle?: ShorthandRenderFunction
+}
+
+/**
+ * Allows users to display data organised in tree-hierarchy.
+ */
+class Tree extends UIComponent<TreeProps> {
+  static create: Function
+
+  static className = 'ui-tree'
+
+  static displayName = 'Tree'
+
+  static propTypes = {
+    ...commonUIComponentPropTypes,
+    ...childrenComponentPropTypes,
+    accessibility: PropTypes.func,
+    items: customPropTypes.collectionShorthand,
+    nested: PropTypes.bool,
```

Comment:
this prop seems to be not used anywhere - am I missing something (please, just reference the line in code in that case)? Thank you

Reply:
I had added a class. Got deleted in refactoring perhaps. Also it might be needed for accessibility props.

Reply:
so, if it is not used anywhere now, lets delete it - even if it will be used for accessibility later, we will add it. Lets limit the API to only those things that are necessary